equipment-register.xsd

```diff
@@ -669,84 +698,85 @@
-                        * &lt;variables&gt;T&lt;/variables&gt;
-                        * &lt;variables&gt;x1,x2,x3&lt;/variables&gt;
+                        See the documentation of [equation](#type_equation) for
+                        more details about the characters that may be used.
+
+                        See the documentation of [variables](#type_variables) for
+                        more details about the `variables` attribute.
                     </xsd:documentation>
                 </xsd:annotation>
-                <xsd:simpleType>
-                    <xsd:restriction base="xsd:string">
-                        <xsd:pattern value="[a-zA-Z]\w*(,[a-zA-Z]\w*)*"/>
-                    </xsd:restriction>
-                </xsd:simpleType>
+                <xsd:complexType>
+                    <xsd:simpleContent>
+                        <xsd:extension base="msl:nonEmptyString">
+                            <xsd:attribute name="variables" type="msl:variables" use="required"/>
+                        </xsd:extension>
+                    </xsd:simpleContent>
+                </xsd:complexType>
             </xsd:element>
             <xsd:element name="uncertainty">
                 <xsd:annotation>
                     <xsd:documentation>
-                        The standard or expanded uncertainty of the corrected value.
+                        The equation to parse to calculate the standard (or expanded)
+                        uncertainty of the corrected value. If the `uncertainty` is a
+                        constant numerical value, set the `variables` attribute value
+                        to be empty (the `variables` attribute must still be defined).
+
+                        See the documentation of [equation](#type_equation) for
+                        more details about the characters that may be used.
+
+                        See the documentation of [variables](#type_variables) for
+                        more details about the `variables` attribute.
                     </xsd:documentation>
                 </xsd:annotation>
-                <xsd:simpleType>
-                    <xsd:restriction base="xsd:float">
-                        <xsd:minInclusive value="0"/>
-                    </xsd:restriction>
-                </xsd:simpleType>
+                <xsd:complexType>
+                    <xsd:simpleContent>
+                        <xsd:extension base="msl:nonEmptyString">
+                            <xsd:attribute name="variables" type="msl:variables" use="required"/>
+                        </xsd:extension>
+                    </xsd:simpleContent>
+                </xsd:complexType>
             </xsd:element>
 			<xsd:element name="coverageFactor">
 				<xsd:annotation>
 					<xsd:documentation>
-						The coverage factor for the standard uncertainty.
+						The coverage factor required to convert the `uncertainty`
+                        to be the standard uncertainty.
```

Comment:
I think it should be 'expanded' uncertainty here, but this depends on the outcome of my previous comment.

Reply:
The `<coverageFactor>` element has been removed.